DEV_SETUP.md

```diff
@@ -413,6 +413,18 @@ that to the new install. If not, proceed to Step 5B.
     ```
 
   - Fire up Fauxton to check that the dbs are there: http://localhost:5984/_utils/
+    - As of CouchDB 3.x, Fauxton is no longer shipped in the container and must
+      be installed separately:
+
+      ```sh
```

Comment:
Should we also add an option to install fauxton globally?

Reply:
I didn't because I prefer not to install project-specific tools globally if I can avoid it. Anyone following these instructions can of course use `-g` if they prefer that. Do you think it would be better to write the instructions to install globally and have nitpicky people like me adjust them to do a local install?

Reply:
Personally, I would be in favor of installing it globally for the sake of convenience but it would be good to give an alternate option too.

We also have recommendations around installing some npm packages globally like `grunt` globally in DEV_SETUP.

Reply:
https://github.com/dimagi/commcare-hq/pull/32529/commits/4acc40a42b53aada4722ae559311626403832cb9

Reply:
Thanks :)